firstvoices/backend/views/search/base_search_views.py

```diff
@@ -79,4 +90,8 @@ def list(self, request, **kwargs):
         # Adding data to objects
         hydrated_objects = hydrate_objects(search_results, request)
 
+        page = self.paginate_queryset(hydrated_objects)
+        if page is not None:
+            return self.get_paginated_response(data=page)
+
```

Comment:
Is this going to scale if someone has 30 pages of search results? I think we'll need to retrieve the page of results directly from elastic search, rather than retrieving extra results and then getting django to paginate them for us

Reply:
I believe we should refrain from altering the length of the list of results outside Elasticsearch, as it may potentially lead to inconsistent outcomes.
For instance, if Elasticsearch returns 25 results for the first page and the user requests only 20 results, omitting this parameter in the ES query could cause Django pagination to remove the last 5 results. Consequently, these 5 results would not be included on the next page either, as they do not exist on Elasticsearch's second page.
One way to avoid this would be to extract the pagination parameters from the request and pass them to the ES query to obtain paginated results directly from Elasticsearch.